contracts/managers/allocation-pricers/BinaryAllocationPricer.sol

```diff
@@ -43,17 +45,16 @@ contract BinaryAllocationPricer is
 {
     using SafeMath for uint256;
 
-    /* ============ Structs ============ */
-    struct CollateralSetInfo {
-        ISetToken setInstance;
-        uint256 componentPrice;
-        uint256 componentDecimals;
-        uint256 collateralValue;
-    }
+    /* ============ Events ============ */
```

Comment:
What other types of allocation-pricers do you think there will be? Can any of this logic in this file be reused?